### ISSUE TITLE
Deleted tracks still exist...

### ISSUE THREAD

**Issue by joeybab3** [kartaview/openstreetcam.org#369]
While checking in to see if the error with iD displaying kartaview tracks had been fixed I noticed that some of my deleted tracks were still present there despite not showing on my profile.

Below is one such example:

https://kartaview.org/details/3511769/0/track-info

It shows as "0 photos" but the slider on the bottom shows 2216 images.

**Comment by joeybab3** [comment]
Hre is another: https://kartaview.org/details/4538105/0/track-info

**Comment by zhaitianduo** [comment]
Will check

**Comment by zhangtiedong0912** [comment]
@joeybab3  hi where are you see the 2216 images? in your profile? could you share a screenshot?

**Comment by joeybab3** [comment]
> @joeybab3 hi where are you see the 2216 images? in your profile? could you share a screenshot?

![image](https://user-images.githubusercontent.com/2829457/164584033-abf1ee0e-adc5-4dcb-8119-5366a9a20aef.png)

**Comment by joeybab3** [comment]
Slider on the bottom of the sequence

**Comment by zhangtiedong0912** [comment]
<img width="1626" alt="image" src="https://user-images.githubusercontent.com/51616101/164586093-050cf566-5abb-4bcf-9445-dec734c44fff.png">
Is it on this page？ i can't see lol @joeybab3

**Comment by joeybab3** [comment]
Yes it's the first link I sent it has the images too, the second 2 do not show the image but show the image count

**Comment by zhangtiedong0912** [comment]
<img width="1661" alt="image" src="https://user-images.githubusercontent.com/51616101/164588254-9a56ad95-5815-4e1c-a096-fe1531093b14.png">
this? @joeybab3

**Comment by joeybab3** [comment]
![image](https://user-images.githubusercontent.com/2829457/164759522-6b931ea3-1919-483b-b1bf-fcfef5533ebb.png)

**Comment by joeybab3** [comment]
Please follow the first link in the original post

**Comment by joeybab3** [comment]
> <img alt="image" width="1661" src="https://user-images.githubusercontent.com/51616101/164588254-9a56ad95-5815-4e1c-a096-fe1531093b14.png">
> 
> this? @joeybab3

For this one though it seems some of the data is still there 
![image](https://user-images.githubusercontent.com/2829457/164759674-f3c7ea66-a4bf-47c1-aa8e-e13a7ebcf5f0.png)
